doc/setup-cmake-environment.md

```diff
@@ -47,8 +75,8 @@ However, some of the dependencies, such as `google-cloud-cpp` are only updated
 monthly. If at some point the Spanner client depends on recent changes on
 `google-cloud-cpp` this strategy will not work for you.
 
-With that out of the way, installing Ninja is recommended, because it makes the
-`vcpkg` builds faster, but not required:
+Though not required, we recommend that you install Ninja as it significantly
+speeds up the `vcpkg` builds:
```

Comment:
We should actually download ninja automatically, so this shouldn't affect vcpkg total runtime. Are you seeing significant perf improvements by pre-installing ninja?

Reply:
I have just been cargo-culting this since forever, thanks for the heads up, I will change the instructions.